src/vendors/candy-machine.service.ts

```diff
@@ -296,9 +302,21 @@ export class CandyMachineService {
       iteration = iteration + 1;
     }
 
+    await this.createHeliusWebhook(candyMachine.address.toBase58());
+
```

Comment:
Helius webhook should never be created within the app itself besides on the endpoint for creating it. We can also add a CLI command for it but unnecessary for now. Please remove it from the CandyMachineService constructor.

What we SHOULD do is just keep it updated with new `accountAddresses` whenever we create a candy machine or an NFT